docs/src/main/paradox/getting-started/running.md

```diff
@@ -60,31 +60,29 @@ Add the `EventGeneratorApp` to your project:
 Scala
```

Comment:
Somewhere in this page it says "Akka Cluster Sharding Typed". Normally we don't include Typed in such module names. Great if you can change to "Akka Cluster Sharding"

Reply:
and the version is wrong for that, not an Akka version:
```
"com.lightbend.akka" %% "akka-cluster-sharding-typed" % "1.0.0-RC1+27-a6e0fa2e"
```

Reply:
I see the groupId is wrong for this and probably a few other modules, I'll doublecheck.  Is that what you're referring to?

The version is generated.

Reply:
it's not using the AkkaVersion, which should be 2.5.8

Reply:
Got it!  Good catch.